models/usage.go

```diff
@@ -33,6 +33,23 @@ func (d DataUsageTier) PricePerGB() float64 {
 	}
 }
 
+// PricePerGBPerHour returns the price per gb per hour
+// we use an approximation of 730 hours
+func (d DataUsageTier) PricePerGBPerHour() float64 {
+	switch d {
+	case Paid:
+		return 0.07 / 730
+	case Partner:
+		return 0.05 / 730
+	case WhiteLabeled:
+		return 0.05 / 730
+	default:
+		// this is a catch all for free tier
+		// free tier users will never encounter a charge call
+		return 9999
```

Comment:
Should't this return 0 for free?

Reply:
Depends I guess. Calling `PricePerGB` or `PricePerGBPerHour` should never be done against a free user and represents incorrect handling of the request. If we went with `0` then it wouldn't be possible to detect this invalid request handling, because the free tier users will always have 0 credits. 

The `9999` is just done to ensure that during development of new functionality, we can detect incorrect handling of a request, since `9999 * X` will always be greater than 0 (because minimum hold time is 1 month and thus X can never be 0) but if we went with `0` instead of `9999` then then it wouldn't be possible to detect this kind of incorrect request handling during development of new features.

Technically we could use 1 or any number greater than 0 really, but the `9999` convention has been something in place for awhile.

Reply:
If it returned 0 for free, the code to use this could be simplified.

Reply:
well the code should *never* use this if it's a free user. That's why you'll see logic in the API where we check the user tier first, and skip these particular calls for free users. If we don't skip the call, an error will due to balance issues that wouldn't be caused if we returned 0 for free.

Basically the process for credit reduction is like this:
* 1) Check user tier, if free skip if not continue
* 2) Determine the cost of the API call
* 3) Ensure user has enough credits to pay for
* 4) Reduce user's available credits


free user processing should skip step 1 to continue on with the request process. If we don't skip step 1, with a `0` being returned here, step 3 check would pass, and then step 4 would also pass as well